src/shared/logs-show.c

```diff
@@ -491,9 +491,15 @@ static int output_short(
                 return 0;
         }
 
+        if (identifier && set_contains(j->exclude, identifier))
+                return 0;
+
         if (!(flags & OUTPUT_SHOW_ALL))
                 strip_tab_ansi(&message, &message_len, highlight_shifted);
 
+        if (flags & OUTPUT_STRIP)
```

Comment:
Also this one. Is it not necessary to apply this to other methods?

Reply:
Same as above